osx/src/platform_osx.mm

```diff
@@ -142,4 +142,6 @@ void cancelUrlRequest(const std::string& _url) {
     }];
 }
 
+void setCurrentThreadPriority(int priority) {}
```

Comment:
`setpriority` is available on OS X too. Unless there was a specific reason for leaving this blank I'll add the same functionality that linux has.